### ISSUE TITLE
More Recent Densepose implementations?

### ISSUE THREAD

**Issue by QiyaoWei** [facebookresearch/DensePose#273]
Hi, 

Correct me if I'm wrong, but I believe caffe has not been actively maintained for a while now (I have also heard that it will soon be merged into pytorch). Therefore, are there any other implementations of this very awesome project in pure pytorch (that doesn't require going through the painful process of installing caffe)? Any help is appreciated. Thanks!

**Comment by vkhalidov** [comment]
@QiyaoWei yes, DensePose is now a part of detectron2: https://github.com/facebookresearch/detectron2/tree/master/projects/DensePose
it's in pure PyTorch.
